tests/test_views.py

```diff
@@ -2520,6 +2520,21 @@ def test_register_ok(self):
         user = User.find_one(Q('username', 'eq', email))
         assert_equal(user.fullname, name)
 
+    def test_register_email_case_insensitive(self):
```

Comment:
When writing a regression test for code that addresses a GH issue, it is useful to include the URL for the issue in a code comment, e.g.

```python
    # Regression test for https://github.com/CenterForOpenScience/osf.io/issues/2902
    def test_register_email_case_insensitive(self):
```

Will add this when I merge.